.prettierignore

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Remove blank line. I removed it locally and `nf-core pipelines lint` didn't complain.

Reply:
cool. passed for me too.